aten/src/ATen/core/Generator.h

```diff
@@ -134,7 +134,7 @@ struct TORCH_API Generator {
 };
 
 template<class Impl, class... Args>
-Generator make_generator(Args&&... args) {
+TORCH_API Generator make_generator(Args&&... args) {
```

Comment:
Doing https://github.com/pytorch/pytorch/pull/93920/files#r1097463936 would allow us to not have to make this an external symbol.